vue.config.js

```diff
@@ -12,6 +12,15 @@ const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
 
 module.exports = {
   configureWebpack: {
+    module: {
+      rules: [
+        {
+          test: /\.mjs$/,
+          include: /node_modules/,
+          type: 'javascript/auto'
+        }
+      ]
+    },
```

Comment:
This is also an experiment as I have not upgraded webpack to 5

Reply:
Obsolete comment?

Reply:
Yes. It is removed now

Reply:
Yes please resolve.